doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -109,18 +109,21 @@ We shall discuss some of the virtual tables in more detail next.
 === Clients Virtual Table
 
 The `clients` virtual table lists all active connections (connected
-clients) including their ip address, port, connection stage, driver
+clients) including their ip address, port, client_options, connection stage, driver
 name, driver version, hostname, protocol version, request count, ssl
 enabled, ssl protocol and user name:
 
 ....
-cqlsh:system_views> select * from system_views.clients;
- address   | port  | connection_stage | driver_name | driver_version | hostname  | protocol_version | request_count | ssl_cipher_suite | ssl_enabled | ssl_protocol | username
------------+-------+------------------+-------------+----------------+-----------+------------------+---------------+------------------+-------------+--------------+-----------
- 127.0.0.1 | 50628 |            ready |        null |           null | localhost |                4 |            55 |             null |       False |         null | anonymous
- 127.0.0.1 | 50630 |            ready |        null |           null | localhost |                4 |            70 |             null |       False |         null | anonymous
+cqlsh> SELECT * FROM system_views.clients;
```

Comment:
nit: I am wondering whether we don't want to make the selection by using `expand on`, not that this is incorrect, just for readability and as a hint to people to use it. WDYT?

Reply:
oh nice suggestion

Reply:
My first thought was to not do so because I want to keep the example on the default settings of cqlsh. But, the idea of having a subtle hint on a less known feature in an example sounds nice. Do you mind when I let the expand on command in the snippet?

Reply:
lgtm